### PR TITLE
TASK: Fix declared dependencies of Neos.Neos

### PR DESCRIPTION
**Pull request by mhsdesign**
requires https://github.com/neos/neos-development-distribution/pull/100

**Upgrade instructions**

**Review instructions**

**Checklist**

- [ ] Code follows the PSR-2 coding style
- [ ] Tests have been created, run and adjusted as needed
- [ ] The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)
- [ ] Reviewer - PR Title is brief but complete and starts with `FEATURE|TASK|BUGFIX`
- [ ] Reviewer - The first section explains the change briefly for change-logs
- [ ] Reviewer - Breaking Changes are marked with `!!!` and have upgrade-instructions

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -35,13 +35,13 @@
         "neos/utility-unicode": "*",
         "doctrine/orm": "^2.6",
         "ext-dom": "*",
-        "neos/twitter-bootstrap": "*",
+        "neos/twitter-bootstrap": "^3.0.6",
```
Comment (ahaeslich) [inline composer.json:35]: Do we really need that here? Didn't we move the dependency to `neos/media-browser`? See: #4673

Comment (ahaeslich) [reply composer.json:35]: Ahhh sorry. I was confused by the (none) package sorting here 🤦

**Review by kitsunet — APPROVED** [review]
Should work now? Will set to automerge

**Comment by kitsunet** [comment]
stan blocking us hard :D

**Comment by mhsdesign** [comment]
Stan will be fixed via #4703

**Pull request merged**